apps/website/.vuepress/theme/global-components/DocIconsPreviewSettings.vue

```diff
@@ -52,8 +52,9 @@ export default {
   grid-template-columns: 180px 240px auto;
   padding: 0.75rem 0;
   position: sticky;
-  top: 133px; // height makes it sticky under search input.
+  top: 131px; // height makes it sticky under search input.
```

Comment:
Why wouldn't we want to use rem here? 6.55rem

Reply:
This was done by me when I implemented the sticky behavior and I forgot to go back and convert to rem instead of starting with rem measurements. Same for the one below. 

@Shijir - if you can add the change 🙏🏼.

Reply:
Thanks! I will make sure to change them to rem before merging in.